ui/raidboss/emulator/data/network_log_converter/LineEvent.ts

```diff
@@ -47,6 +49,7 @@ export default class LineEvent {
 
   prefix(): string {
     const timeString = DTFuncs.timeToTimeString(this.timestamp, this.tzOffsetMillis, true);
+    // TODO: should raidemulator not convert lines that don't come from the ffxiv plugin?
```

Comment:
FYI @valarnin

Reply:
I think it's a bit more complicated than that, because to replicate the behavior exactly as it current works in ACT, we'd have to also not emit log line events for the converted versions of lines. We'd need to make the converted line optional, then if it's undefined during playback, skip it for emitting lines.

Reply:
Yeah, my thoughts were along those same lines.  I think it is ok for now as no regexes should match, but maybe something to clean up later.